docs/manual/host-configuration/index.md

```diff
@@ -108,7 +108,7 @@ Airtime's media monitor has its own configuration file in */etc/airtime/media-mo
 Playout and recorder settings
 -----------------------------
 
-Settings for pypo, the playout and recording engine used by Airtime, are found in the file */etc/airtime/pypo.cfg*. After making changes to this file, you will have to issue the command:
+Settings for pypo, the playout and recording engine used by Libretime, are found in the file */etc/airtime/pypo.cfg*. After making changes to this file, you will have to issue the command:
```

Comment:
These settings have been moved to `/etc/airtime/airtime.conf`

Reply:
Will be in next PR.